src/storage/txn/latch.rs

```diff
@@ -149,7 +149,7 @@ impl Latches {
     where
         H: Hash,
     {
-        let mut s = DefaultHasher::new();
+        let mut s = FxHasher::default();
```

Comment:
Have you found that the hash here has much impact on the system? I think we'd better be cautious of replacing it with FxHasher because it cannot prevent attacks for hash collisions. I am not sure whether it is possible to construct specific keys to slow down the whole system.

Reply:
I found the widely used HashMap (defined in components/tikv_util/src/collections.rs) was based on the FxHasher, seems the Latch has similar feature with HashMap, so I guess the FxHasher may also suitable for the Latch. (Maybe Latch need to ensure lower hash collisions, 
 but so does HashMap, so I wonder why we use FxHasher in HashMap?)

Reply:
In most HashMap usages in tikv, the keys are allocated by ourselves. So it is not a problem as there are no malcious collision attacks. The keys here can be constructed by end users. That's why we should be cautious.

Reply:
In fact, TiKV authors probably didn't think much about such a security issue. I believe there can be some HashMap usages in tikv now when FxHash actually shouldn't be used.

Reply:
Ok, I get it, thanks! I'll reuse the DefaultHasher.

Reply:
> there can be some HashMap usages in tikv now when FxHash actually shouldn't be used.

Can you name some?

Reply:
@BusyJay Never mind. I've gone through every place a Hasher is used and confirmed there isn't a case when the key can be easily derived from the end user and a unsafe Hash function is used. Really sorry for the unsound blame.

Reply:
I firstly thought the it was vulnerable to use a unsafe hash function when doing aggregation, but soon realized the database cannot prevent it. It should be the application who is responsible for it.